.circleci/config.yml

```diff
@@ -542,11 +542,13 @@ jobs:
     steps:
       - checkout
       - *restore_cache_step
-      - run: tox -e 'requests_contrib-{py27,py34,py35,py36}-requests{208,209,210,211,212,213,219}' --result-json /tmp/requests.results
+      - run: tox -e 'requests_contrib-{py27,py34,py35,py36}-requests{208,209,210,211,212,213,219}' --result-json /tmp/requests.1.results
```

Comment:
I think you can do `'requests_contrib{,_autopatch}-{...'`

Reply:
Indeed. @labbati is there a reason we have the other auto-patch tests separated out?

Reply:
They can be unified now. The reason used to be that for autopatch we needed to dist and install while for the other tests we did not (to go faster). Now (after https://github.com/DataDog/dd-trace-py/pull/654) we do not need the "dist -> install" anymore and we should unify them (when possible).

Reply:
I'll leave it for this PR and open up a PR to clean them up.